x-pack/test/security_solution_api_integration/README.md

```diff
@@ -3,42 +3,30 @@
 ## Subdirectories
-
-1. `config` stores base configurations specific to both the Serverless and ESS environments, These configurations build upon the base configuration provided by `xpack/test_serverless` and `x-pack-api_integrations`, incorporating additional settings such as environment variables and tagging options.
-
-
-2. `test_suites` directory now houses all the tests along with their utility functions. As an initial step,
-we have introduced the `detection_response` directory to consolidate all the integration tests related to detection and response APIs.
-
+- `config` stores base configurations specific to both the Serverless and ESS environments, These configurations build upon the base configuration provided by `xpack/test_serverless` and `x-pack-api_integrations`, incorporating additional settings such as environment variables and tagging options.
+- `es_archive` and `es_archive_path_builder` directories contain the data that can be used by the tests
+- `scripts` directory contains various scripts used to run the tests
+- `test_suites` directory houses all the tests along with their utility functions. As an initial step, we have introduced the `detection_response` directory to consolidate all the integration tests related to detection and response APIs.
 
 ## Overview
 
-- In this directory, Mocha tagging is utilized to assign tags to specific test suites and individual test cases. This tagging system enables the ability to selectively apply tags to test suites and test cases, facilitating the exclusion of specific test cases within a test suite as needed.
-
-- Test suites and cases are prefixed with specific tags to determine their execution in particular environments or to exclude them from specific environments. 
-
-- We are using the following tags:
-   * `@ess`: Runs in an ESS environment (on-prem installation) as part of the CI validation on PRs.
-
-   * `@serverless`: Runs in the first quality gate and in the periodic pipeline.
-
-   * `@serverlessQA`: Runs in the Kibana QA quality gate.
-
-   * `@skipInEss`: Skipped for ESS environment.
-
-   * `@skipInServerless`: Skipped for all quality gates, CI and periodic pipeline.
-
-   * `@skipInServerlessMKI`: Skipped for all the MKI environments. 
-
-ex:
-```
- describe('@serverless @ess create_rules', () => { ==> tests in this suite will run in both Ess and Serverless
-   describe('creating rules', () => {}); 
-
-  // This test is skipped due to flakiness in serverless environments: https://github.com/elastic/kibana/issues/497777
-   describe('@skipInServerless missing timestamps', () => {}); ==> tests in this suite will be excluded in Serverless
-
- ```
+Test suites and cases are prefixed with specific tags (see [this docs link](https://docs.elastic.dev/security-solution/teams/analyst-experience/automation/serverless/2.labeling.mdx) for more information on the tags we are using) to determine their execution in particular environments or to exclude them from specific environments.
```

Comment:
My only concern is if we are allowed to share internal links in a public repo.

Reply:
good point, I brought back the list of labels! Thanks for catching this!